pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.7.0"
+    version: "2.6.1"
```

Comment:
Is there a problem with upgrading to the version 2.7.0?

Reply:
Newer versions, listed in 'Latest', may not be mutually compatible.